### PR TITLE
added a index page for blogs

### LINKED ISSUE #86: As a guest I want to see all the blogs from all SDGs on one page so I can have a over view on all the blog what is there
Acceptance Criteria:
GIVEN On the home page
WHEN Click on a button
THEN Show all the blogs on one page.

### PR DESCRIPTION
**Pull request by Koreyoshi159**
Quick fix for separate SDG blog pages,
Added a page for all the blogs.

### CODE REVIEW

**Review by AlexGit123 — APPROVED** [review]

File: resources/views/sdg.blade.php [hunk 1]
```diff
@@ -27,31 +27,32 @@
...
-                        </div>
-                        <div class="subtitle is-6 sdg-content">
-                            Impact: {{ $blog->impact }}<br>
-                            Research Group: <br>
-                            Program: {{ $blog->program->name }}<br>
-                            Business Operation: {{ $blog->businessOperation->name }}<br>
-                            Subgoal: {{ $blog->subSdgs }}<br>
-                            Publisher: {{ $blog->contact_name }} <br>
-                            Updated at: {{ $blog->updated_at }} <br>
+            @continue($blog->visibility == 0)
```
Comment (awqfrxx) [inline resources/views/sdg.blade.php:30]: how does this work? the @continue

**Review by awqfrxx — APPROVED** [review]

File: resources/views/sdg.blade.php (hunk 1 above)
Comment (Koreyoshi159) [reply resources/views/sdg.blade.php:30]: From what I know is If it is 0 then it goes to the next one but how it exactly works idk xD

**Review by avanheeren — APPROVED** [review]

**Review by koenhamstra — CHANGES_REQUESTED** [review]
I do not see anything in the blog page beside the text "All blogs on one page
Use the filters to search for what you want".

**Review by koenhamstra — APPROVED** [review]
Looks good and it works

**Pull request merged**